lib/transaction/operation/create_account.go

```diff
@@ -53,6 +53,10 @@ func (o CreateAccount) GetAmount() common.Amount {
 	return o.Amount
 }
 
-func (o CreateAccount) HasFee() bool {
+// If isSourceLinked is true, tx.Source is frozen account.
+func (o CreateAccount) HasFee(isSourceLinked bool) bool {
+	if isSourceLinked || o.Linked != "" {
```

Comment:
A node cannot trust the client.
please remove o.Linked check

Reply:
o.Linked is used to check create frozen account.